packages/grid/src/GridMetricCalculator.ts

```diff
@@ -1234,10 +1690,10 @@ class GridMetricCalculator {
 
   /**
    * Sets the width for the specified column
-   * @param {Number} column The column model index to set
-   * @param {Number} size The size to set it to, or null to reset the column size
+   * @param column The column model index to set
+   * @param size The size to set it to, or null to reset the column size
    */
-  setColumnWidth(column, size) {
+  setColumnWidth(column: ModelIndex, size?: number): void {
```

Comment:
Comment says "or null to reset". Types defined here wouldn't allow that (would allow undefined or omitted)

Actually the logic says setting to null would delete the width from the map

Reply:
Yea it was a loose null check, but was passing `undefined` in. I'll update to clarify.